pkg/virt-api/rest/subresource.go

```diff
@@ -566,7 +566,7 @@ func (app *SubresourceAPIApp) StartVMRequestHandler(request *restful.Request, re
 }
 
 func (app *SubresourceAPIApp) StopVMRequestHandler(request *restful.Request, response *restful.Response) {
-	// RunStrategyHalted         -> doesn't make sense
+	// RunStrategyHalted         -> force stop if graceperiod=0, otherwise doesn't make sense
```

Comment:
Can you adjust this comment with the info that the graceperiod should be shorter that the actual, please? Thanks